x-pack/plugins/infra/server/routes/snapshot/lib/apply_metadata_to_last_path.ts

```diff
@@ -56,6 +56,10 @@ export const applyMetadataToLastPath = (
           lastPath.ip = ipAddresses;
         }
       }
+      if (inventoryFields.os) {
+        const inventoryFieldsOs = get(firstMetaDoc, inventoryFields.os) as string;
```

Comment:
nit: we could maybe shorten this? 
```suggestion
        lastPath.os = get(firstMetaDoc, inventoryFields.os) as string;
```